src/test/java/cloud/fogbow/common/util/SystemUserUtilTest.java

```diff
@@ -20,7 +25,30 @@ public void testSerializationDeserealizationOfSystemUserObject() throws Unexpect
         Assert.assertEquals(systemUser, recoveredSystemUser);
     }
 
+    // Test if size of content is going to overflow the limit a SystemUSer is allowed
```

Comment:
Standardize the test by following the example:
    // test case: ...
    @Test(expected = Exception.class) // verify
    public void test() {
        // set up
        // exercise
    }